Source/Behaviors/BrickLayoutBehavior.swift

```diff
@@ -10,42 +10,42 @@ import Foundation
 
 /// A BrickBehavior can alter the way bricks are displayed and handled
 /// By subclassing the Behavior, an implementation can alter the frame of the brick as well as its origin
-public class BrickLayoutBehavior: NSObject {
-    public internal(set) weak var brickFlowLayout: BrickFlowLayout?
+open class BrickLayoutBehavior: NSObject {
+    open internal(set) weak var brickFlowLayout: BrickFlowLayout?
 
     // Flag that indicates that this behavior needs some calculation down stream to function correctly
-    public var needsDownstreamCalculation: Bool {
+    open var needsDownstreamCalculation: Bool {
         return false
     }
 
-    public func shouldUseForDownstreamCalculation(for indexPath: NSIndexPath, with identifier: String, forCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> Bool {
+    open func shouldUseForDownstreamCalculation(for indexPath: IndexPath, with identifier: String, forCollectionViewLayout collectionViewLayout: UICollectionViewLayout) -> Bool {
```

Comment:
`open func shouldUseForDownstreamCalculation(for indexPath: IndexPath, with identifier: String, for collectionViewLayout: UICollectionViewLayout) -> Bool {`